sklearn/feature_selection/tests/test_info_gain_ratio.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tests for info_gain_ratio
```

Comment:
I don't think this requires a separate file, especially since the tests are almost identical with those currently present for IG.